test/integration/next/default-pages-dir/test/next-build.test.ts

```diff
@@ -116,11 +106,19 @@ test("next build works", async () => {
   expect(bunBuild.exitCode).toBe(0);
 
   const bunCliOutput = (await new Response(bunBuild.stdout).text())
+    // normalize bytes
+    .replace(/\d(?:\.\d+)?(?= k?B)/g, "0")
```

Comment:
if we are disabling parts of this test, please add a todo comment to fix the bug that is causing these byte differences.

there is no reason for these to actually be different file sizes, but it isn't this PR's responsibility to fix

Reply:
Reverted, but left normalization commented out for reference.